terraform/bod_bastion_security_group_rules.tf

```diff
@@ -34,3 +34,93 @@ resource "aws_security_group_rule" "bastion_ssh_to_docker" {
   from_port = 22
   to_port = 22
 }
+
+# Allow all ICMP from vulnscanner instance in Management VPC,
+# for internal scanning
+resource "aws_security_group_rule" "bod_bastion_ingress_all_icmp_from_mgmt_vulnscan" {
+  count = "${var.enable_mgmt_vpc_access_to_all_vpcs}"
+
+  security_group_id = "${aws_security_group.bod_bastion_sg.id}"
+  type = "ingress"
+  protocol = "icmp"
+  cidr_blocks = [
+    "${aws_instance.mgmt_nessus.private_ip}/32"
+  ]
+  from_port = -1
```

Comment:
Why port -1?

Reply:
To close the loop on this, when the protocol is ICMP, -1 for `from_port` and `to_port` means "allow all types of ICMP".  I cannot find any official documentation on this, but a web search turns up various posts that mention it, like this:  https://blog.jwr.io/terraform/icmp/ping/security/groups/2018/02/02/terraform-icmp-rules.html